spec/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "jasmine": true
```

Comment:
No need 'cause we are not using `waitsForPromise` anymore, we're using `async` `it`s now